libs/@local/harpc/client/typescript/src/codec/Decoder.ts

```diff
@@ -42,6 +42,7 @@ export interface Decoder<E = DecodingError, R = never>
   };
 }
 
+// eslint-disable-next-line @typescript-eslint/no-empty-object-type
```

Comment:
Maybe not worth doing but could set a rule in this package:

```
{
    rules: {
      "@typescript-eslint/no-empty-object-type": ["error", { allowInterfaces: "with-single-extends" }]
    },
}
```

Reply:
didn't know it had that option, good shout!

added in [ee2cac6](https://github.com/hashintel/hash/pull/5847/commits/ee2cac6e84c5058c9f8a7f14d2f14827cc914b14)